cmd/hashets/hashets.go

```diff
@@ -69,6 +70,7 @@ func init() {
 			return nil
 		})
 	flag.BoolVar(&replace, "replace", false, "delete the original original files after hashing")
+	flag.BoolVar(&codeOnly, "code-only", false, "generate code only without copying files")
```

Comment:
I think `map-only` would be a bit more expressive.

```suggestion
	flag.BoolVar(&mapOnly, "map-only", false, "generate map file only without copying files")
```